hazelcast/reactor.py

```diff
@@ -42,7 +42,7 @@ def _loop(self):
         Future._threading_locals.is_reactor_thread = True
         while self._is_live:
             try:
-                asyncore.loop(count=1, timeout=0.1, map=self._map)
```

Comment:
I don't think we should set the timeout to 0. When it is 0, this becomes basically a busy loop with a select system call on each iteration. When the timeout is 0.1, 0.01 or 0.001 idle client uses %0-0.1 CPU on my laptop. When I set timeout to 0, CPU usage jumps to %18.  Also, setting the timeout to 0 slightly increased the latency on my machine. I think 0.1 or 0.01 would be a good value for the timeout.

Reply:
You are right, I didn't see this on my desktop. I played with a few timeouts, and yes, sub-millis performs equally well. Changed it to 0.01.